select.dust

```diff
@@ -1,4 +1,4 @@
-<div class="c-select {class} {?label}c--inner-label{/label}" data-stencil="stencil-select">
+<div class="c-select {class} {?label}c--inner-label{/label}" data-adaptivejs-component="stencil-select">
```

Comment:
Yes – although this is a bit weird and could throw people off – it's not doing what you might expect it to do – instead it’s doing what it *must* do to make the underlying html work. In a perfect world, we might have some way to have the id on the outside be the one you provided, and fake the inner id, but that would be pretty complicated and might be just as bad. We just need time to figure this stuff out.